plugins/rebalance/utils.go

```diff
@@ -0,0 +1,108 @@
+package rebalance
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/gluster/glusterd2/glusterd2/store"
+	rebalanceapi "github.com/gluster/glusterd2/plugins/rebalance/api"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	rebalancePrefix string = "rebalance/"
+)
+
+// TimeVal represents the time value
+type TimeVal struct {
```

Comment:
You don't really need a seperate type here as it isn't used else where.
You can do it with local variables in the setCommitHash function.

Reply:
Done